bin/git-plop

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+find_plop_json() {
+  current_dir=$(pwd)
+
+  while [ "$current_dir" != "/" ]; do
+    if [ -f "$current_dir/.plop.json" ]; then
+      echo "$current_dir/.plop.json"
+      return 0
+    fi
+    current_dir=$(dirname "$current_dir")
+  done
+
+  echo "Error: .plop.json not found"
+  return 1
+}
+
+plop_json_path=$(find_plop_json)
+
+if [ "$plop_json_path" == "Error: .plop.json not found" ]; then
+  echo "$plop_json_path"
+  exit 1
+fi
+
+plop_json_dir=$(dirname "$plop_json_path")
+
+src=$(jq -r '.src' "$plop_json_path")
+workdir=$(jq -r '.workdir' "$plop_json_path")
+
+src_dir="$plop_json_dir/$src"
+workdir_dir="$plop_json_dir/$workdir"
+
+echo "Source directory: $src_dir"
+echo "Working directory: $workdir_dir"
+
+if [ ! -f "$plop_json_dir/package.json" ]; then
```

Comment:
wrong condition here. you have to search files in src directory, not where .plop.json is located.

Change logic of info/error messages as follows:

1) check src_dir has package.json AND  (plopfile.ts OR plopfile.js). In case not, print:
`Unable to find plopfile at [{src_dir}] `
exit 1

2) Print info:
`Found plopfile: [path to plopfile].`
`Run plop with codebase: [path to workdir]`

run docker